### PR TITLE
fix: Sda 1899

### PR DESCRIPTION
**Pull request by johankwarnmarksymphony**
https://perzoinc.atlassian.net/browse/sda-1899 Add shortcuts between 1.5, 2.0-stable and 2.0-daily

### CODE REVIEW

File: src/app/window-handler.ts [hunk 1]
```diff
@@ -1035,6 +1042,114 @@ export class WindowHandler {
...
     }
 
+    /**
+     * HACK SWITCH to Client 1.5
+     */
+    private async onClient1_5(): Promise <void> {
+        logger.info('window handler: go to Client 1.5');
+        logger.info('this.globalConfig.url: ' + config.getGlobalConfigFields([ 'url']));
+
+        if (config.getGlobalConfigFields([ 'url']).url && config.getGlobalConfigFields([ 'url']).url.startsWith('https://corporate.symphony.com')) {
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: Please move this to a const

File: src/app/window-handler.ts [hunk 2]
```diff
@@ -1035,6 +1042,114 @@ export class WindowHandler {
...
+            logger.info('window handler: switchClient: ' + switchClient);
+            if (switchClient === 'client1_5' ) {
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: I think it's better to rename as currentClient

File: src/app/window-handler.ts [hunk 3]
```diff
@@ -1035,6 +1042,114 @@ export class WindowHandler {
...
+            }
+
+            switchClient = 'client1_5';
+
+            const focusedWindow = BrowserWindow.getFocusedWindow();
+
+            const dogfoodUrl = `https://corporate.symphony.com/`;
+            if (focusedWindow && windowExists(focusedWindow)) {
+                await focusedWindow.loadURL(dogfoodUrl);
+                reloadWindow(focusedWindow as ICustomBrowserWindow);
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: Why do we need to reload after loadURL here?

File: src/app/window-handler.ts [hunk 4]
```diff
@@ -1035,6 +1042,114 @@ export class WindowHandler {
...
+        }
+    }
+
+    /**
+     * HACK SWITCH to Client Mana-stable
+     */
+    private async onClientManaStable(): Promise <void> {
+        logger.info('window handler: go to Client Mana-stable');
+
+        if (config.getGlobalConfigFields([ 'url']).url && config.getGlobalConfigFields([ 'url']).url.startsWith('https://corporate.symphony.com')) {
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: Same as above

File: src/app/window-handler.ts [hunk 5]
```diff
@@ -1035,6 +1042,114 @@ export class WindowHandler {
...
+            }
+        } else {
+            logger.info('Switch not support for this POD-url');
+        }
+    }
+
+    /**
+     * HACK SWITCH to Client Mana-daily
+     */
+    private async onClientManaDaily(): Promise <void> {
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: We can completely remove this function and use the above function to load the URL dynamically

File: src/app/window-handler.ts [hunk 6]
```diff
@@ -967,6 +979,15 @@ export class WindowHandler {
         globalShortcut.register(isMac ? 'Cmd+Alt+I' : 'Ctrl+Shift+I', this.onRegisterDevtools);
         globalShortcut.register('CmdOrCtrl+R', this.onReload);
 
+        // Hack to switch between Client 1.5, Mana-stable and Mana-daily
+        if (checkIfCorporate()) {
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]:
```suggestion
        if (this.globalConfig.url && this.globalConfig.url.startsWith('https://corporate.symphony.com')) {
```

File: src/app/window-handler.ts [hunk 7]
```diff
@@ -46,6 +46,18 @@ export interface ICustomBrowserWindow extends Electron.BrowserWindow {
...
 
+enum ClientVersionTypes {
+    CLIENT_1_5 = 'client_1_5',
+    CLIENT_MANA_STABLE = 'client_mana_stable',
+    CLIENT_MANA_DAILY = 'client_mana_daily',
+}
+
+let currentClient = ClientVersionTypes.CLIENT_1_5;
+
+const checkIfCorporate = () => {
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: we can remove this now

File: src/app/window-handler.ts (hunk 2 above)
Comment (johankwarnmarksymphony) [reply src/app/window-handler.ts]: done

File: src/app/window-handler.ts (hunk 4 above)
Comment (johankwarnmarksymphony) [reply src/app/window-handler.ts]: done

File: src/app/window-handler.ts (hunk 7 above)
Comment (johankwarnmarksymphony) [reply src/app/window-handler.ts]: done

**Pull request merged**